terraform/organizations-accounts-cica.tf

```diff
@@ -4,7 +4,7 @@ locals {
 }
 
 resource "aws_organizations_account" "cica" {
-  name      = "CICA"
+  name      = "CICA Shared Services"
```

Comment:
Can you confirm you've followed the [How do I change the name on my AWS account?](https://aws.amazon.com/premiumsupport/knowledge-center/change-organizations-name/) guide for this - you'll need to login to the AWS root user for this AWS account.

Reply:
Have forwarded that info thanks

Reply:
You'll need to remove the name change in this PR until the above is completed.